tldw_Server_API/app/core/config_sections/moderation.py

```
@@ -0,0 +1,260 @@
+) -> str:
+    if env_key is not None:
+        env_value = env_map.get(env_key)
+        if env_value is not None and str(env_value).strip() != "":
+            return str(env_value)
+
+    raw = config_parser.get("Moderation", option, fallback=default)
+    text = str(raw).strip()
+    return text or default
+
+
+def _parse_bool(raw: object, default: bool) -> bool:
```

Ruff (0.15.11)

[warning] Boolean-typed positional argument in function definition

(FBT001)